pkg/core/blockchain.go

```diff
@@ -1062,8 +1062,8 @@ func (bc *Blockchain) UnsubscribeFromExecutions(ch chan<- *state.AppExecResult)
 // amount of NEO between specified blocks. The amount of NEO being passed is in
 // its natural non-divisible form (1 NEO as 1, 2 NEO as 2, no multiplication by
 // 10⁸ is neeeded as for Fixed8).
-func (bc *Blockchain) CalculateClaimable(value int64, startHeight, endHeight uint32) util.Fixed8 {
-	var amount util.Fixed8
+func (bc *Blockchain) CalculateClaimable(value int64, startHeight, endHeight uint32) int64 {
+	var amount int64
```

Comment:
Some `CalculateClaimable` user are probably need to be updated too now, but I don't see that in 3d04fd106d02ec8701f2740c6add15961cb2ead9.